custom_components/lock_code_manager/entity.py

```diff
@@ -85,7 +85,9 @@ def __init__(
 
         self._attr_name: str | None = " ".join(key_parts)
         self._attr_unique_id = f"{self.base_unique_id}|{slot_num}|{key}"
-        self._attr_extra_state_attributes = {ATTR_CODE_SLOT: int(slot_num)}
+        self._attr_extra_state_attributes: dict[str, Any] = {
```

Comment:
It's really a dict[str, int | list[str]]. If a code slot is disabled, the active binary sensor will include a list of keys that are causing the binary sensor to be off. So if it's because of the calendar, it will be [calendar]. There may be multiple conditions that are failing which is why it's a list. Not sure how yet but it seemed like a potentially useful thing to include

Reply:
Ah, I see. Still, it would probably be best to annotate more tightly and `dict[str, Any]` that way any accidental extra types aren't stuck in there...